plugin/src/main/java/io/jenkins/plugins/analysis/warnings/axivion/AxivionSuite.java

```diff
@@ -385,10 +404,10 @@ public ListBoxModel doFillCredentialsIdItems(
                 }
             }
             return result.includeAs(
-                    ACL.SYSTEM,
-                    item,
-                    StandardUsernamePasswordCredentials.class,
-                    Collections.emptyList())
+                            ACL.SYSTEM,
```

Comment:
Auto-formatting of IntelliJ with the checked-in code styles.

Reply:
BTW: these ACL elements are deprecated, any chance to update these in a followup PR?

Reply:
Yes, I can take a look.
The doc suggests using `ACL.SYSTEM2` but it does seem to be another api.